internal/kafka/internal/workers/clusters_mgr.go

```diff
@@ -179,6 +179,7 @@ func (c *ClusterManager) Reconcile() []error {
 		c.processProvisioningClusters,
 		c.processProvisionedClusters,
 		c.processReadyClusters,
+		c.processWaitingForKasFleetshardOperatorClusters,
```

Comment:
```suggestion
		c.processWaitingForKasFleetshardOperatorClusters,
		c.processReadyClusters,
```
let's process waiting for kas-fleet-shard operator clusters first